SoftCam.Key

```diff
@@ -2,6 +2,945 @@
+P 0015 0060AF19 723F737A57370E ;EMM Key 
+P 0015 00612C35 A800B5CF633602 ;EMM Key 
+P 0015 00612C62 F5003FD8BE08A2 ;EMM Key
+---------------------------------------------
+P 00150002 00 56596630737AE2 ;BYU TV, 15W, 11568 H
+P 00150002 01 5D52F62E082733 ;BYU TV, 15W, 11568 H
+P 0015 00648E11 D8E6985B24D02C ;EMM Key
+---------------------------------------------
+P 015200CD 00 65CAA4ABC614BC ;EU Swe - Fin, 15W, 11568 H
+P 015200CD 01 5D52F62E082733 ;EU Swe - Fin, 15W, 11568 H
+P 015200D4 00 65CAA4ABC614BC ;EU: Arm - Bul, 15W, 11568 H
+P 015200D4 01 5D52F62E082733 ;EU: Arm - Bul, 15W, 11568 H
+P 015200CF 00 65CAA4ABC614BC ;EU: Dut - Man, 15W, 11568 H
+P 015200CF 01 5D52F62E082733 ;EU: Dut - Man, 15W, 11568 H
+P 015200D3 00 65CAA4ABC614BC ;EU: Efi - Ara, 15W, 11568 H
+P 015200D3 01 5D52F62E082733 ;EU: Efi - Ara, 15W, 11568 H
+P 015200D6 00 65CAA4ABC614BC ;EU: Eng - Fre, 15W, 11568 H
+P 015200D6 01 5D52F62E082733 ;EU: Eng - Fre, 15W, 11568 H
+P 015200C9 00 65CAA4ABC614BC ;EU: Eng - Ger, 15W, 11568 H
+P 015200C9 01 5D52F62E082733 ;EU: Eng - Ger, 15W, 11568 H
+P 015200CB 00 65CAA4ABC614BC ;EU: Eng - Ita, 15W, 11568 H
+P 015200CB 01 5D52F62E082733 ;EU: Eng - Ita, 15W, 11568 H
+P 015200D5 00 65CAA4ABC614BC ;EU: Eng - Por, 15W, 11568 H
+P 015200D5 01 5D52F62E082733 ;EU: Eng - Por, 15W, 11568 H
+P 015200CE 00 65CAA4ABC614BC ;EU: Eng - Rus, 15W, 11568 H
+P 015200CE 01 5D52F62E082733 ;EU: Eng - Rus, 15W, 11568 H
+P 015200D7 00 65CAA4ABC614BC ;EU: Eng - Spa, 15W, 11568 H
+P 015200D7 01 5D52F62E082733 ;EU: Eng - Spa, 15W, 11568 H
+P 015200D2 00 65CAA4ABC614BC ;EU: Est - Ukr, 15W, 11568 H
+P 015200D2 01 5D52F62E082733 ;EU: Est - Ukr, 15W, 11568 H
+P 015200D0 00 65CAA4ABC614BC ;EU: Hun - Rom, 15W, 11568 H
+P 015200D0 01 5D52F62E082733 ;EU: Hun - Rom, 15W, 11568 H
+P 015200CC 00 65CAA4ABC614BC ;EU: Nor - Dan, 15W, 11568 H
+P 015200CC 01 5D52F62E082733 ;EU: Nor - Dan, 15W, 11568 H
+P 015200D1 00 65CAA4ABC614BC ;EU: Pol - Alb, 15W, 11568 H
+P 015200D1 01 5D52F62E082733 ;EU: Pol - Alb, 15W, 11568 H
+P 015200CA 00 65CAA4ABC614BC ;EU: Spa - Por, 15W, 11568 H
+P 015200CA 01 5D52F62E082733 ;EU: Spa - Por, 15W, 11568 H
+P 015200D9 00 65CAA4ABC614BC ;EU: Far - Lat, 15W, 11568 H
+P 015200D9 01 5D52F62E082733 ;EU: Far - Lat, 15W, 11568 H
+P 015200D8 00 65CAA4ABC614BC ;EU: Gre - Cze, 15W, 11568 H
+P 015200D8 01 5D52F62E082733 ;EU: Gre - Cze, 15W, 11568 H
+P 0152 00648E11 D8E6985B24D02C ;EMM Key
+---------------------------------------------
+P 002200C9 00 FEE998ABBE894D ;CNN Newsource Europe 22W
+P 002200C9 01 842F2801182286 ;CNN Newsource Europe 22W
+P 0022 0041BB3E 285A68374F0AC1 ;EMM Key
+---------------------------------------------
+P 003000CC 00 191F1B713B6777 ;Agencia EFE, 30W, 12563 H
+P 003000CC 01 1D1BE37186BEA0 ;Agencia EFE, 30W, 12563 H
+P 003000CD 00 191F1B713B6777 ;EFE Radio, 30W, 12563 H
+P 003000CD 01 1D1BE37186BEA0 ;EFE Radio, 30W, 12563 H
+P 0030 005B5030 8FF6FBD0AF7218 ;EMM Key
+---------------------------------------------
```

Comment:
P 016E004D 00 0123456789ABCD ;SNAI TV Audio Unico, 16E, 12699 H
P 016E004D 01 97D6968EEC34A3 ;SNAI TV Audio Unico, 16E, 12699 H
P 016E 004E9002 339D50F5A6EB4D ;EMM Key 
P 016E 004F2AB5 47A1C66FE78F9A ;EMM Key 
P 016E 0052B933 52CE687BDA3265 ;EMM Key
---------------------------------------------
P 016E0065 00 CF8F8EBA3286A4 ;101-IPPICA1, 16E, 12724 H
P 016E0065 01 8C0993635C22EA ;101-IPPICA1, 16E, 12724 H
P 016E0066 00 CF8F8EBA3286A4 ;102-IPPICA2, 16E, 12724 H
P 016E0066 01 8C0993635C22EA ;102-IPPICA2, 16E, 12724 H
P 016E 0050E5A1 814EAED71CC782 ;EMM Key
P 016E 00528BF2 FBDF2752B5A9E1 ;EMM Key
P 016E 0053B631 7AB8E5D9F5F5C0 ;EMM Key
---------------------------------------------
P 00090066 00 9CB7875316982A ;AFN|prime Atlantic, 9.0E 11804 V
P 00090066 01 32C8BE9499BC34 ;AFN|prime Atlantic, 9.0E 11804 V
P 00090067 00 9CB7875316982A ;AFN|news, 9.0E 11804 V
P 00090067 01 32C8BE9499BC34 ;AFN|news, 9.0E 11804 V
P 00090068 00 9CB7875316982A ;AFN|sports, 9.0E 11804 V
P 00090068 01 32C8BE9499BC34 ;AFN|sports, 9.0E 11804 V
P 00090069 00 9CB7875316982A ;AFN|Prime Pasific, 9.0E 11804 V
P 00090069 01 32C8BE9499BC34 ;AFN|Prime Pasific, 9.0E 11804 V
P 0009006A 00 9CB7875316982A ;AFN|spectrum, 9.0E 11804 V
P 0009006A 01 32C8BE9499BC34 ;AFN|spectrum, 9.0E 11804 V
P 0009006B 00 9CB7875316982A ;AFN|sports 2, 9.0E 11804 V
P 0009006B 01 32C8BE9499BC34 ;AFN|sports 2, 9.0E 11804 V
P 0009006C 00 9CB7875316982A ;AFN|family, 9.0E 11804 V
P 0009006C 01 32C8BE9499BC34 ;AFN|famil, 9.0E 11804 V
P 0009006D 00 9CB7875316982A ;AFN|movie, 9.0E 11804 V
P 0009006D 01 32C8BE9499BC34 ;AFN|movie, 9.0E 11804 V
P 00090078 00 9CB7875316982A ;AFN Atlantic, 9.0E 11804 V (radio)
P 00090078 01 32C8BE9499BC34 ;AFN Atlantic, 9.0E 11804 V (radio)
P 00090079 00 9CB7875316982A ;The Voice, 9.0E 11804 V (radio)
P 00090079 01 32C8BE9499BC34 ;The Voice, 9.0E 11804 V (radio)
P 0009007A 00 9CB7875316982A ;Power Talk, 9.0E 11804 V (radio)
P 0009007A 01 32C8BE9499BC34 ;Power Talk, 9.0E 11804 V (radio)
P 0009007B 00 9CB7875316982A ;Clutch Sports Radio, 9.0E 11804 V (radio)
P 0009007B 01 32C8BE9499BC34 ;Clutch Sports Radio, 9.0E 11804 V (radio)
P 0009007C 00 9CB7875316982A ;AFN Fans, 9.0E 11804 V (radio)
P 0009007C 01 32C8BE9499BC34 ;AFN Fans, 9.0E 11804 V (radio)
P 0009007D 00 9CB7875316982A ;AFN Hot AC, 9.0E 11804 V (radio)
P 0009007D 01 32C8BE9499BC34 ;AFN Hot AC, 9.0E 11804 V (radio)
P 0009007E 00 9CB7875316982A ;AFN Legacy, 9.0E 11804 V (radio)
P 0009007E 01 32C8BE9499BC34 ;AFN Legacy, 9.0E 11804 V (radio)
P 0009007F 00 9CB7875316982A ;AFN Joe FM, 9.0E 11804 V (radio)
P 0009007F 01 32C8BE9499BC34 ;AFN Joe FM, 9.0E 11804 V (radio)
P 00090080 00 9CB7875316982A ;AFN Gravity, 9.0E 11804 V (radio)
P 00090080 01 32C8BE9499BC34 ;AFN Gravity, 9.0E 11804 V (radio)
P 00090081 00 9CB7875316982A ;AFN Country, 9.0E 11804 V (radio)
P 00090081 01 32C8BE9499BC34 ;AFN Country, 9.0E 11804 V (radio)
P 00090082 00 9CB7875316982A ;NPR International, 9.0E 11804 V (radio)
P 00090082 01 32C8BE9499BC34 ;NPR International, 9.0E 11804 V (radio)
P 00090083 00 9CB7875316982A ;AFN Seasonal, 9.0E 11804 V (radio)
P 00090083 01 32C8BE9499BC34 ;AFN Seasonal, 9.0E 11804 V (radio)
P 00090084 00 9CB7875316982A ;AFN Contingency, 9.0E 11804 V (radio)
P 00090084 01 32C8BE9499BC34 ;AFN Contingency, 9.0E 11804 V (radio)
P 00090084 01 9CB7875316982A ;Freedom Radio, 9.0E 11804 V (radio)
P 00090085 01 32C8BE9499BC34 ;Freedom Radio, 9.0E 11804 V (radio)
P 00090087 00 9CB7875316982A ;Aviano Eagle, 9.0E 11804 V (radio)
P 00090087 01 32C8BE9499BC34 ;Aviano Eagle, 9.0E 11804 V (radio)
P 00090088 00 9CB7875316982A ;Bavaria Eagle, 9.0E 11804 V (radio)
P 00090088 01 32C8BE9499BC34 ;Bavaria Eagle, 9.0E 11804 V (radio)
P 00090089 00 9CB7875316982A ;Benelux Eagle, 9.0E 11804 V (radio)
P 00090089 01 32C8BE9499BC34 ;Benelux Eagl, 9.0E 11804 V (radio)
P 0009008A 00 9CB7875316982A ;Incirlik Eagle, 9.0E 11804 V (radio)
P 0009008A 01 32C8BE9499BC34 ;Incirlik Eagle, 9.0E 11804 V (radio)
P 0009008B 00 9CB7875316982A ;Kaiserslautern Eagle, 9.0E 11804 V (radio)
P 0009008B 01 32C8BE9499BC34 ;Kaiserslautern Eagle, 9.0E 11804 V (radio)
P 0009008C 00 9CB7875316982A ;Naples Eagle, 9.0E 11804 V (radio)
P 0009008C 01 32C8BE9499BC34 ;Naples Eagle, 9.0E 11804 V (radio)
P 0009008D 00 9CB7875316982A ;Rota Eagle, 9.0E 11804 V (radio)
P 0009008D 01 32C8BE9499BC34 ;Rota Eagle, 9.0E 11804 V (radio)
P 0009008E 00 9CB7875316982A ;Souda Bay Eagl, 9.0E 11804 V (radio)
P 0009008E 01 32C8BE9499BC34 ;Souda Bay Eagle, 9.0E 11804 V (radio)
P 0009008F 00 9CB7875316982A ;Spangdahlem Eagle, 9.0E 11804 V (radio)
P 0009008F 01 32C8BE9499BC34 ;Spangdahlem Eagle, 9.0E 11804 V (radio)
P 00090090 00 9CB7875316982A ;Stuttgart Eagle, 9.0E 11804 V (radio)
P 00090090 01 32C8BE9499BC34 ;Stuttgart Eagle, 9.0E 11804 V (radio)
P 00090091 00 9CB7875316982A ;Vicenza Eagle, 9.0E 11804 V (radio)
P 00090091 01 32C8BE9499BC34 ;Vicenza Eagle, 9.0E 11804 V (radio)
P 00090092 00 9CB7875316982A ;Weisbaden Eagle, 9.0E 11804 V (radio)
P 00090092 01 32C8BE9499BC34 ;Weisbaden Eagle, 9.0E 11804 V (radio)
P 00090093 00 9CB7875316982A ;Bahrain, 9.0E 11804 V (radio)
P 00090093 01 32C8BE9499BC34 ;Bahrain, 9.0E 11804 V (radio)
P 00090094 00 9CB7875316982A ;Sembach Afternoon Radio, 9.0E 11804 V (radio)
P 00090094 01 32C8BE9499BC34 ;Sembach Afternoon Radio, 9.0E 11804 V (radio)
P 0009 0053F2AC ADFC9B2781CB6B ;EMM Key
P 0009 00540716 FCCD8E6C12268A ;EMM Key
P 0009 005C379D D374275FBF4A0F ;EMM Key
P 0009 005CDAEF FA1A2056882210 ;EMM Key
---------------------------------------------
---------------------------------------------
P 040500FC 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500FC 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500FE 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500FE 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500D2 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500D2 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500DC 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500DC 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500E6 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500E6 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500F0 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500F0 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 040500FA 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 040500FA 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 0405012B 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 0405012B 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 04050100 00 891C1C5B7EEACA ;FOX BRASIL HD, 40.5 W, 4095
P 04050100 01 E1161571C63F58 ;FOX BRASIL HD, 40.5 W, 4095
P 0405 005EA08F 6F10F91A97228F ;EMM Key
P 0405 0062A585 54F0459E1A11FC ;EMM Key
P 0405 00536B20 737B8CAEA064AD ;EMM Key
P 0405 005CC879 6EE3843DF06DC3 ;EMM Key
P 0405 005C14C7 DED4F2BEE33CD7 ;EMM Key
P 0405 00534008 8CB53697E79D95 ;EMM Key
P 0405 00597E18 90A01B9E0B69B6 ;EMM Key
P 0405 00531FEC 205029EA50E12C ;EMM Key
P 0405 005C77E0 CC7ECD162BEB9C ;EMM Key
---------------------------------------------
P 040600C8 00 0A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 040600C8 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 0406006D 00 3A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 0406006D 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 040600C9 00 3A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 040600C9 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 04060070 00 3A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 04060070 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 04060071 00 3A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 04060071 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 0406006F 00 3A063A84E24613 ;ESPN Brasil HD, 40.5 W, 3980
P 0406006F 01 3D0202842C9D3B ;ESPN Brasil HD, 40.5 W, 3980
P 04060072 00 3A063A84E24613 ;FOX Sports, 40.5 W, 3980
P 04060072 01 3A063A84E24613 ;FOX Sports, 40.5 W, 3980
P 04060073 00 3A063A84E24613 ;FOX Sports 2, 40.5 W, 3980
P 04060073 01 3A063A84E24613 ;FOX Sports 2, 40.5 W, 3980
P 0406 0059EDF7 BBD81262FF1DAE ;EMM Key
P 0406 005ABEB6 E5A2C6D58A6839 ;EMM Key
P 0406 0059A85E 7D3FEBFAA6D97F ;EMM Key 
P 0406 005989D0 91E46C51440990 ;EMM Key 
P 0406 005E9EFF 5633358C2FO0F9 ;EMM Key
---------------------------------------------
P 04080064 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080064 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408012C 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408012C 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080020 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080020 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 040800C8 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 040800C8 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408002B 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408002B 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080028 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080028 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408001F 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408001F 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080011 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080011 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080010 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080010 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408000D 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408000D 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080168 00 CF47D18EBABE08 ;FS ARGANTINA HD, 40.5 W, 3730
P 04080168 01 A5C4E052191DF6 ;FS ARGANTINA HD, 40.5 W, 3730
P 0408 0061DD5D D74666BB5C8A60 ;EMM Key
---------------------------------------------
P 0407025C 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025C 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 04070262 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 04070262 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 04070259 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 04070259 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025A 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025A 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025B 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025B 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025D 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025D 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025E 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407025E 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 040702BB 00 CF47D18EBABE08 ;FOX COLOMBIA HD, 40.5 W, 3808
P 040702BB 01 A1B2C316D4E5F8 ;FOX COLOMBIA HD, 40.5 W, 3808
P 0407 004D5A03 70EB084B5AADD8 ;EMM Key
---------------------------------------------
P 0409005E 00 D8674FF934185F ;TelefeHD 40.5W
P 0409005E 01 1444E5F66A55DB ;TelefeHD 40.5W
P 040900C6 00 D8674FF934185F ;FUTURE
P 040900C6 01 1444E5F66A55DB ;FUTURE
P 040900C5 00 D8674FF934185F ;MTV 00s HD
P 040900C5 01 1444E5F66A55DB ;MTV 00s HD
P 040900C1 00 D8674FF934185F ;TeenNick Global HD
P 040900C1 01 D8674FF934185F ;TeenNick Global HD
P 040900C4 00 D8674FF934185F ;UKMTV LIVE
P 040900C4 01 D8674FF934185F ;UKMTV LIVE
P 0409 0060CD47 F6D68A0F90B3D0 ;EMM Key
P 0409 0060D094 E093D99F0E5BE9 ;EMM Key
P 0409 00626890 86D1FC3A54E806 ;EMM Key
P 0409 005EE2DF 5E846B53DD6D64 ;EMM Key
P 0409 0059418C E36989D6E7B0AF ;EMM Key 
---------------------------------------------
P 00A2006A 00 0123456789ABCD; 40w 3723 AMC Brasil 40w (3723R)
P 00A20068 00 0123456789ABCD; 40w 3723 AMC Colombia
P 00A2006B 00 0123456789ABCD; 40w 3723 AMC Mï¿½xico
P 00A20065 00 0123456789ABCD; 40w Arts
P 00A20067 00 0123456789ABCD; 40w 3723 Pelï¿½culas y artes Mï¿½xico HD
P 00A20069 00 0123456789ABCD; 40w 3723 Sundance Brasil
P 00A203ED 00 0123456789ABCD ; Added on 2023-08-24 21:33:44.670593 for ESPN Andina at 40.6W
P 00A203EE 00 0123456789ABCD ; Added on 2023-08-24 21:34:39.115293 for ESPN Peru at 40.6W

---------------------------------------------
P 0409 0061DD5D D74666BB5C8A60 ;40.5W 3728 
P 0409 0062A585 54F0459E1A11FC ;40.5W 4096
P 4040FFFF 00 CF47D18EBABE08 ;ECM Fox Peru 3803 H 30000
P 4040FFFF 01 A5C4E052191DF6 ;ECM Fox Peru 3803 H 30000
P 05E87F32 GROUP 4040 ; Fox Peru 3803 H 30000
P 5040FFFF 00 0123456789ABCD ;ECM AMC 3723 R 30000
P 5040FFFF 01 E42DB81767E02E ;ECM AMC 3723 R 30000 
P 094F1E47 GROUP 5040 ; AMC 3723 R 30000
P 6040FFFF 00 0123456789ABCD ;ECM AMC Latinoamerica 3763 R 30000
P 6040FFFF 01 24C9C28DB78D3C ;ECM AMC Latinoamerica 3763 R 30000 
P 8944A75D GROUP 6040 ; AMC Latinoamerica 3763 R 30000
P 9040FFFF 00 56DBB98DBA634D ;ECM Wobi-TV 4041 V
P 9040FFFF 01 2B3ECB0234CB71 ;ECM Wobi-TV 4041 V 
P B883EF98 GROUP 9040 ; Wobi-TV 4041 V
P 9040 001A075C 68577698407D66 ; 40.5W emm
P 9040 004D599A 31580D1F2C537C ; EMM
P 1140FFFF 00 24C9C28DB78D3C ;ECM Fox Life Latin America 3728 L 19300
P 1140FFFF 01 821529F13C6111 ;ECM Fox Life Latin America 3728 L 19300
P TBD GROUP 1140 ; Fox Life Latin America 3728 L 19300 
P 1140 0061DD5D D74666BB5C8A60 ; EMM 3728 L 19300
P 1240FFFF 00 FF51A9E7CCEC11 ; ECM Venevisionn 4177 R 5416
P 1240FFFF 01 CCA329286A1437 ; ECM Venevisionn 4177 R 5416
P 3C872DE4 GROUP 1240 ; Venevisionn 4177 R 5416
P 1240 001A075C 68577698407D66 ; EMM 
P 1240 005E9E93 A78B992C6C86DB ; EMM
P 1240 005D1896 339DDA82ADB558 ; EMM
P 1240 0061DD5D D74666BB5C8A60 ; EMM
P 1240 0062A585 54F0459E1A11FC ; EMM
P 4010013A 00 2F6B46F9FED00F0 ;SYFY/SU BRASIL HD;4047V(R)5900
P 4010013A 01 6549B5F085DB29 ;SYFY/SU BRASIL HD;4047V(R)5900
P 40100138 00 2F6B46F9FED00F ;SYFY/SU BRASIL HD;4047V(R)5900
P 40100138 01 6549B5F085DB29 ;SYFY/SU BRASIL HD;4047V(R)5900
P 4010 61DD5D D74666BB5C8A6000 ;EMM Key
P 4004FFFF  00 0123456789ABCD ; EUROPA HD amc 3964R30000 
P 4004FFFF  01 E42DB81767E02E ; EUROPA HD amc 3964R30000
P 8944A75D GROUP 4004 ; EUROPA HD 3964R30000
P 4011FFFF 00 FF51A9E7CCEC11 ; VENEVISION
P 4011FFFF 01 CCA329286A1437 ; VENEVISION
P 70038539 GROUP 4011 ; VENEVISION
---------------------------------------------

P 00010015 00 2E15845426EA6F ; GOLTV ALT at 40.6W
P 0001000D 00 2E15845426EA6F ; GOLTV HD at 40.6W
P 00010013 00 2E15845426EA6F ; GOLTV UY at 40.6W
P 0001000F 00 2E15845426EA6F ; VTV+ HD at 40.6W
P 0001000B 00 2E15845426EA6F ; VTV URUGUAY HD at 40.6W
P 00010011 00 2E15845426EA6F ; GOLTV ECUADOR at 40.6W
P 00010063 00 2E15845426EA6F ; OS Download at 40.6W
P 00010015 01 2E15845426EA6F ; GOLTV ALT at 40.6W
P 0001000D 01 2E15845426EA6F ; GOLTV HD at 40.6W
P 00010013 01 2E15845426EA6F ; GOLTV UY at 40.6W
P 6001000F 01 2E15845426EA6F ; VTV+ HD at 40.6W
P 0001000B 01 2E15845426EA6F ; VTV URUGUAY HD at 40.6W
P 00010011 01 2E15845426EA6F ; GOLTV ECUADOR at 40.6W
P 00010063 01 2E15845426EA6F ; OS Download at 40.6W

P 0001 005D1896 339DDA82ADB558; EMM Key Gol TV, VTV 40.5Wâ4061 R 20000
P 0001 005E9E93 A78B992C6C86DB ; EMM Key Gol TV, VTV 40.5W-4061 R 20000
P 0001 005D1896 339DDA82ADB558; EMM Key Gol TV, VTV 40.5W-4061 R 20000
---------------------------------------------
P 1043FFFF 00 719C60F09B91D0 ; 	TNT SPORTS AC3 3800 H 30000 43w
P 1043FFFF 01 A29A663DFB8B95 ; 	TNT SPORTS AC3 3800 H 30000 43w
P D5CD49BD GROUP 1043 ; 	    TNT SPORTS AC3 3800 H 30000 43w
P A2CA792B GROUP 1043 ; 	    TNT SPORTS AC3 3800 H 30000 43w
P 1043 0062538C D760E50F903DC3 ; EMM Key TP 3800 H 30000 43w
P 1043FFFF 00 B0855ADAF355E7 ; added by Emu Mon Jan  6 13:02:20 2020 UA: 0062538C
P 1043FFFF 01 10B555F75F4A4E ; added by Emu Mon Jan  6 13:02:20 2020 UA: 0062538C
P 2043FFFF 00 CF8F8EBA3286A4 ; 	CC NORTE 43w 3927 
P 2043FFFF 01 3AB2AAFA1AB2D2 ; 	CC NORTE 43w 3927 
P 9AE6DA8D GROUP 2043 ; 	    CC NORTE 43w 3927 
P EDE1EA1B GROUP 2043 ; 	    CC NORTE 43w 3927
P 3043FFFF 00 CF8F8EBA3286A4 ; 	CINECANAL MEX HD, 43.0W, 3895
P 3043FFFF 01 3AB2AAFA1AB2D2 ; 	CINECANAL MEX HD, 43.0W, 3895
P 6C1A8055 GROUP 3043 ; 	    CINECANAL MEX HD, 43.0W, 3895
P 1B1DB0C3 GROUP 3043 ; 	    CINECANAL MEX HD, 43.0W, 3895
P 4043FFFF 00 CF8F8EBA3286A4 ; 	43w 3717 V  Fox Comedy HD
P 4043FFFF 01 3AB2AAFA1AB2D2 ; 	43w 3717 V  Fox Comedy HD
P 1C2CD1B8 GROUP 4043 ; 	    43w 3717 V  Fox Comedy HD
P 6B2BE12E GROUP 4043 ; 	    43w 3717 V  Fox Comedy HD
P 5043FFFF 00 1E0DB7452A4446 ; 	HD TEATHER HD SPA, 43.1W, 3744V
P 5043FFFF 01 2E4E26E501106F ; 	HD TEATHER HD SPA, 43.1W, 3744V
P 1B12152E GROUP 5043 ; 	    THD TEATHER HD SPA, 43.1W, 3744V
P 6043FFFF 00 CF8F8EBA3286A4 ; 	CNN Mexico, 43.0W, 4040
P 6043FFFF 01 3AB2AAFA1AB2D2 ; 	CNN Mexico, 43.0W, 4040
P 41DE7509 GROUP 6043 ; 	    CNN Mexico, 43.0W, 4040
P D1616898 GROUP 6043 ; 	    CNN Mexico, 43.0W, 4040
P 7043003C 00 6AB47BE65525CB ;Playboy HD, 43.0W, 3973
P 7043003C 01 5EDA5D1E725DD6 ;Playboy HD, 43.0W, 3973
P 8043FFFF 00 0123456789ABCD ; 	Venus, 43.0W, 4079
P 8043FFFF 01 0123456789ABCD ; 	Venus, 43.0W, 4079
P C0BCB92C GROUP 8043 ; 	    Venus, 43.0W, 4079
P 9043FFFF 00 00000000000000 ; 	TyCSHD, 43.0W, 4053 y 4026 V 43w
P 9043FFFF 01 00000000000000 ; 	TyCSHD, 43.0W, 4053 y 4026 V 43w
P EE570817 GROUP 9043 ; 	    TyCSHD, 43.0W, 4053 y 4026 V 43w
P 99503881 GROUP 9043 ; 	    TyCSHD, 43.0W, 4053 y 4026 V 43w
P C456FD52 GROUP 9043 ; 	    Canal 13 Europa 4026 V 43w
P 5D5FACE8 GROUP 9043 ; 	    Canal 13 Europa 4026 V 43w
P 9043 005A48D4 AF82076F1E8C1D ; AUTORROLL EMM Key 4053
P 9043FFFF 00 28CD93EC91161A ; added by Emu Mon Jan  6 12:50:29 2020 UA: 005A48D4
P 9043FFFF 01 0B135B5748D674 ; added by Emu Mon Jan  6 12:50:29 2020 UA: 005A48D4
P 1143FFFF 00 8FBBAD67BCA0F5 ; 	Animal Planet Latin HD 3967 43w
P 1143FFFF 01 70300730D9E16A ; 	Animal Planet Latin HD 3967 43w
P 5899E12A GROUP 1143 ; 	    Animal Planet Latin HD 3967 43w
P 2F9ED1BC GROUP 1143 ; 	    Animal Planet Latin HD 3967 43w
P 1143 0050E478 0C26DF4A8F3700 ; EMM Key TP-3966-H-21090
P 1143 004F73CD E3276FE6BBE4E3 ; EMM Key [Valid, EMM_Intel_11_43w_TP-3967-H-21089.bin]
P 1143 0051D06B 37B0000D6F6214 ; EMM Key [Valid, EMM_Intel_11_43w_TP-3967-H-21089.bin]
P 1243FFFF 00 191F1B713B6777 ; 	CINEMA DINAMITA, 3863 H 17140 43w
P 1243FFFF 01 1D1BE37186BEA0 ; 	CINEMA DINAMITA, 3863 H 17140 43w
P A8406BA0 GROUP 1243 ; 	    CINEMA DINAMITA, 3863 H 17140 43w
P 38FF7631 GROUP 1243 ; 	    CINEMA DINAMITA, 3864 H 17140 43w
P 1243 00650579 17FE74B893D917 ; EMM Key TP-3863-H-17142
P 1243 0061A957 6E0415308D42E8 ; EMM Key TP-3863-H-17142
P 1243 00631934 E1F638B81F3421 ; EMM Key TP-3863-H-17142
P 1243 0054F076 2ACB3A7F808DDA ; EMM Key TP-3863-H-17142
P 1343FFFF 00 56DBB98DBA634D ; 	Disney HD Lat, 43.1W, 3772
P 1343FFFF 01 2B3ECB0234CB71 ; 	Disney HD Lat, 43.1W, 3772
P 342E6926 GROUP 1343 ; 	    Disney HD Lat, 43.1W, 3772
P AA4AFC85 GROUP 1343 ; 	    Disney HD Lat, 43.1W, 3772
P 1443FFFF 00 0F1BD579C4312E ; 	Brazil Bloomberg 4086 V 7500 43w
P 1443FFFF 01 12179D780F8856 ; 	Brazil Bloomberg 4086 V 7500 43w
P C00BE558 GROUP 1443 ; 	    Brazil Bloomberg 4086 V 7500 43w
P 1543FFFF 00 0123456789abcd ; 	SexyHot 3980 V 2500 43w
P 1543FFFF 01 0123456789abcd ; 	SexyHot 3980 V 2500 43w
P 0BEC68CB GROUP 1543 ; 	    SexyHot 3980 V 2500 43w
#############################################
#            Group 47.5w channels             #
############################################# 
P 1047FFFF 00 2A3D509FCDE4EC ; ECM 47.5w 4055 R 30000
P 1047FFFF 01 B797AB0815B2FA ; ECM 47.5w 4055 R 30000
P F6898172 GROUP 1047 ; 47.5w 4055 R 30000
p 1047 0054527B DBAEE0A2A2B170 ; EMM 47.5w 4055 R 30000
P 04750005 00 AD9BBE01E61A06 ; Platino HD 3965R10416 
P 04750005 01 403569443EE2B2 ; Platino HD 3965R10416 
P 04750006 00 AD9BBE01E61A06 ; Platino HD 3965R10416 
P 04750006 01 403569443EE2B2 ; Platino HD 3965R10416
P 0475001D 00 AD9BBE01E61A06 ; Platino HD 3965R10416 
P 0475001D 01 403569443EE2B2 ; Platino HD 3965R10416
---------------------------------------------
P 0008040E 00 4BF83DA12B5414 ;FEED OCCAS HD, 0.8W, 12643 V
P 0008040E 01 63923DEC3B12AC ;FEED OCCAS HD, 0.8W, 12643 V
P 0008 004D95EA 9EF08988DD74C8 ;EMM Key 
P 0008 00502ACC 2BDE65982BCAF1 ;EMM Key 
P 0008 004F8849 4550B157D61DBF ;EMM Key